CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.18.0
+  * Migrate to null safety.
+
 ## 0.17.11-dev
```

Comment:
`0.17.11-dev` hasn't been published yet; you can either add the null safety changelog line to the `0.17.11-dev` list - and plan to bump the version later, before publishing - or add it here but rename the entry to 0.18.0.

Reply:
We should also make sure this version matches what's in the pubspec. I'd keep both as `0.17.11-dev` for now for simplicity.